content/rest/versions/v1.0/channels.textile

```diff
@@ -26,7 +26,7 @@ Ably Platform organizes the message traffic within applications into named chann
 
 h2(#getting-started). Getting started
 
-The Ably REST client library provides a straightforward API for "publishing":/rest/messages#publish messages and retrieving "message history":/api/rest-sdk/channels#history from a channel.
+The Ably REST client library provides a straightforward API for "publishing":/api/rest-sdk/channels#publish messages and retrieving "message history":/api/rest-sdk/channels#history from a channel.
```

Comment:
Should this be `/message`?

Reply:
Publish is a channel method. So it's documented in the channels.textile file.

Reply:
Should have explained what I meant better. I meant to say that in this instance (the getting started section), should we be linking to the message page to explain what a message is rather than directly to the method? Out of scope of this change though maybe, as it's correcting a broken link already.